lib/topological_inventory/schema/default.rb

```diff
@@ -123,6 +123,63 @@ def add_tagging_collection(model, manager_ref: [:source_ref])
         end
       end
 
+      def add_service_instances
+        add_collection(:service_instances) do |builder|
+          add_default_properties(builder)
+          add_default_values(builder)
+
+          save_block = lambda do |source, inventory_collection|
+            src_refs = inventory_collection.data.collect { |inventory_object| inventory_object.source_ref }
+            return if src_refs.blank?
+
+            # Saving Service Instances as usual
+            InventoryRefresh::SaveCollection::Base.send(:save_inventory, inventory_collection)
```

Comment:
@slemrmartin I think a better way to do this would be to have another collection e.g. `service_instance_tasks` and define service_instances as a dependent collection